lib/danger/danger_core/environment_manager.rb

```diff
@@ -7,6 +7,8 @@ class EnvironmentManager
 
     # Finds a Danger::CI class based on the ENV
     def self.local_ci_source(env)
+      return Danger::LocalOnlyGitRepo if LocalOnlyGitRepo.validates_as_ci? env
```

Comment:
regression: There should be a way to not fall into this if. I lost the capability to simulate my workflows locally that depend on the gitlab API, and the same will happen for other SCMs =/

In my scenario, I run `bundle exec danger staging --pry` with some gitlab envs such `GITLAB_CI`, etc.

Reply:
Upon reviewing the code, this MR makes much more sense than relying on CI load ordering.

My workaround for local simulation is to wrap the pry as it happens in the staging command, and run it as simple danger execution with the CI variables correctly set. Sorry for the confusion 😅